translate/storage/test_lisa.py

```diff
@@ -0,0 +1,38 @@
+from lxml import etree
+
+from translate.storage.lisa import LISAfile
+
+class TestCleanIndent:
+    def _xmlfromstring(self, xmlstring):
+        """TODO: return etree document constructed from passed xmlstring"""
```

Comment:
You can probably remove that `TODO` over here, was mostly left out from my comment because my methods had all that storage handling from the usual storage tests and I wanted to put the focus on the actual test methods themselves.